### DISCUSSION TITLE
2023-05-05 - May the 4th be with you - Chewing the Fat 📈 Automation ⭐

### DISCUSSION THREAD

**Discussion by lukecookssw** [SSWConsulting/SSW.Rules.Content#5494] [Chewing the Fat]
Greetings Jedi, Sith Lords, and Padawans.

Let's talk about [this rule](https://ssw.com.au/rules/automation-essentials)

**Comment by PothieuG** [comment]
I believe that automation brings real comfort to work and can also help prevent injuries. 
As long as we still retain the option of manual action, I am 100% convinced.

**Comment by tiagov8** [comment]
Best way to show the benefits of automation is by providing the experience. It's awesome SSW offices are great showcases of this service!

**Comment by brydeno** [comment]
The real challenge of automation is making it as reliable as a manual process.

**Comment by leahy268** [comment]
Huge fan of automation. I haven't had much experience with the SSW automation yet though.  It needs to be super reliable though as the Sys Admin in me always says what happens to the front door if the computer running it crashes.

Great video though guys.

**Comment by leehawkinsssw** [comment]
I thought I was the only person in the world who hasn't seen "Star Wars", so thanks Adam for making me feel slightly less of a freak.

The idea of equipping my home with so-called "smart" devices accessible over the internet fills me with horror. There are so many security and privacy concerns around these devices and many have been found to have serious issues in both departments. All this really means is a swag of questionable software running on questionable hardware, bringing with it the inevitable maintenance, upgrade and obsolescence issues plaguing any IT system. 

I'll be sticking to keys and switches for now. #oldschool

**Comment by matt-goldman** [comment]
I think the rule nails it. It starts by acknowledging that automation is cool, but then gives some very practical examples of how it can actually improve your safety or comfort at work.

**Comment by danielmackay** [comment]
Another cracking video @lukecookssw and team!

I can see the value for remotely controlling office/home spaces (i.e. checking lights/AC is off, or starting them remotely on a hot day before arriving home).  Voice activation could also be cool for controlling these sorts of things.

**Comment by SamBWagner** [comment]
@lukecookssw The one whom we can rely on for fun videos. Keep em' coming. 

Automation is awesome! We just don't leverage it that much in Brisbane.

**Comment by tkapa** [comment]
Cool rule. Been wanting to do home automation for a while now.

**Comment by jakebayliss** [comment]
I like the example given, makes a lot of sense to make the unlock/lock process easy. Hard part seems to be maintaining everything

**Comment by bahjatalaadelssw** [comment]
The Rule makes a strong point about simplifying several steps. I agree that with more components comes future maintenance and cost. However, being able to connect with your home remotely to manage functions sounds useful.

**Comment by ozdemir-mehmet** [comment]
Expanding on Brydren's comment:
Automated processes have the potential to greatly enhance efficiency and productivity, but to be successful, they must be made as reliable as manual processes. This requires careful planning, testing, and maintenance to ensure that the system can handle unexpected events and errors, as well as the flexibility to adapt and evolve over time. Proper employee training is also essential to ensure that staff can use and troubleshoot the automated system effectively. By prioritizing reliability, we can reap the benefits of automation without sacrificing the quality and consistency of their processes.

**Comment by MattParkerDev** [comment]
The SSW Rule does a good job of outlining reasons to implement automation in the office. I think some downsides should be listed, and steps we take or can be taken to mitigate them.

**Comment by AlexRothwell** [comment]
Automation is great but of course involves extra moving parts, each of which can break so it's important to also continue maintaining a manual ways of doing things. This gives you all the benefits of the automation with the peace of mind that if something breaks you can still do everything
An example of this is smart lighting: when it loses power or network connectivity, it reverts to working as a normal light

**Comment by sethdaily** [comment]
My favourite automations are the temperature control in Sydney, and the automatic lights in the bathrooms.

**Comment by chrisschultzssw** [comment]
Temperature is a very useful thing to automate - and while it can be near-impossible to retrofit, a full building management system with automatic louvres, shades, etc. can greatly reduce energy usage.

**Comment by Levijj22** [comment]
Great video! Perfect from a sales point of view! Fun and intriguing!
I want everything automated....
The more tech the better, who cares if "they" are listening 😆 I'm not that interesting!

**Comment by bradystroud** [comment]
Light switches are better than apps

**Comment by tombui99** [comment]
Automation is cool and all but it's still a long way there before it becomes as reliable and affordable as manual processes.

**Comment by MarcusIrmscherSSW** [comment]
I am all for anything that helps reduce cognitive load, especially when it comes to security. Not having to remember security passes or keys is very helpful.

**Comment by cvionnet** [comment]
Nice exemple from the rule to show how automation can help daily life. And solution like Control4 really helps.
The video is priceless!
![image](https://user-images.githubusercontent.com/46954368/236350198-ca19ee25-3c94-41ce-b7e5-2304ac6af65d.png)

**Reply by jimmy1992abc** [reply]
> Nice exemple from the rule to show how automation can help daily life. And solution like Control4 really helps. The video is priceless! ![image](https://user-images.githubusercontent.com/46954368/236350198-ca19ee25-3c94-41ce-b7e5-2304ac6af65d.png)

Legendary

**Comment by ncn-ssw** [comment]
Rules regarding redundant modes of operation for automation would be useful.

**Comment by jimmy1992abc** [comment]
Can we change the setting of the light in the bathroom 
From
            Motion Sensor
To
            Sensor in the door lock

I need to keep moving my body while I am dumping something!!!

**Reply by tomek-i** [reply]
Free exercise

**Comment by JackDevAU** [comment]
Another great video by Luke!! 
Now for me to setup some automation at home 🥳

**Comment by UlyssesMaclaren** [comment]
It makes us look techy when clients come in... that's the main win for me :)

Also... the video was classic! well done to Luke and the TV team 😅🤣

**Comment by camillars** [comment]
Love the video, @lukecookssw and SSW TV! This was a perfect marketing-nerdy opportunity to create content.
I love the office automation, it really makes a difference here in the Sydney office :)

**Comment by zacharykeeping** [comment]
Fun video! I have a decent smart home setup at home and having all my lights and heaters automated has become essential to me. The main issue though is when guests come over and get confused with how to control smart devices, so I think having automations while still having reliable physical controls is the way to go.

**Comment by jontygardner** [comment]
Automation is awesome ⭐  
RIP to the doorman - however, they provide a human connection that makes people feel comfortable 

One thing to note: please don't take away the opportunity to still manually flick a light on or off. It can be frustrating when the automation or the automation app is not working and there is no way to switch a light off or on physically. Imagine going into a sauna and there was no emergency button or latch you could press because the sauna is 'automated' so nothing to worry about 😉

**Comment by jackreimers** [comment]
I think automation in a work setting makes a lot of sense as there is usually someone who manages maintaining the systems, and it provides a lot of benefits. I'm not sold on home automation though as it can lead to network and even physical security issues depending on the products you use and whether they are kept up to date and can be a major chore if devices or apps stop working.

**Reply by tomek-i** [reply]
There are lot of videos on youtube of security researches exposing those flaws. Many devices or apps have 0 security and I don't like the idea of having the control software hosted externally by the vendor. It's something I want to run locally and expose it locally - preferably on a separate, private network and not connect everything to my wifi network.

I haven't looked into it yet but there are some open source options of control software :-D 

I also agree that automation should be complementary to the manual process and not replace it.

**Comment by amankumarrr** [comment]
The video was awesome with a clear msg on how automation empowers us to feel superhuman, would like to see the Melbourne office to be truly automated like this.

**Comment by KahaMason** [comment]
Video was hilarious! 😄 The automation in the office has definitely inspired me to step up my game for automation at home.

**Comment by JackDuan123** [comment]
I like this video

**Comment by calumjs** [comment]
Funny video.
I'd like to work on a GPT integration that will be aware of sensor inputs and be able to chat to users (including proactive messages and responses). It will convert the users intent from the chat into Control4 instructions. Along the lines of this example I mentioned: https://www.youtube.com/live/jndjad5GSAs?feature=share&t=6628

**Reply by tomek-i** [reply]
Jarvis, turn on the lights :-P

**Comment by Abf09** [comment]
The idea of automation seems attractive, not sure how reliable it is. Sometimes it does not work and its time consuming to get it to work manually. I'm all up for it if we have products working efficiently 🙂

**Comment by JeanThirion** [comment]
Automation is cool, I think we're overdoing it a bit but it's fine and fun.
Also big Wow factor when clients walk in so I guess it's worth it

**Comment by drwharris** [comment]
Great video !

As far as automation ... Automating the heating/cooling and lighting to save on power?

**Comment by Ben0189** [comment]
This is a great video👍

Nice to have automation, but I think we're overdoing it (eg the shade)

**Comment by jeoffreyfischer** [comment]
The rule is very clear and and concise. And the automation of most features makes it an easier-to-use environment and reduces the potential stress of unautomated workplaces.

**Comment by AshAnil** [comment]
Great video Luke ⭐
Automation is fantastic. It greatly increases the efficiency, accuracy, and speed of repetitive tasks, freeing up time for more valuable work.

**Comment by Hona** [comment]
Awesome video Luke! Shared it as soon as I saw it last night

**Comment by Marxoz** [comment]
Love the video, even though I am not a Star Wars fan. Luke has some serious acting skills.

**Comment by steven0x51** [comment]
Automation control is a cool thing, but sometimes I just wanted to adjust the brightness of a light, I have to go through the following steps:
1. unlock my phone (potentially take a second try or even have to enter the PW)
2. find the app
3. wait for it to connect to the system
4. browse to the room
5. browse to the control
6. adjust the brightness.

If we could mount some control panels on the wall in each section of the office, we are still able to use our phone to control, but we can optionally skip steps 1-4 if we are close to the control panels.

**Comment by christoment** [comment]
👋 This is not the comment that you are looking for 👋

Just kidding :)
One thing to take care about is the risk management strategy for automation.
E.g. is it fail-safe or fail-close? What secure fallback mechanism to have on out of power scenario?

Because the security is as good as the weakest link in the chain :)

**Comment by wicksipedia** [comment]
Ah, young one. Your pursuit of knowledge in the realm of automation is commendable. But, as Emperor Palpatine, let me provide you with some insights that may further guide your journey towards absolute control and efficiency.

Title: "Do you automate your essentials?"

A more suitable title for your piece would be, "Automate Your Essentials or Face the Consequences."

Content:

Introduction:

Your feeble attempt at an introduction lacks the true essence of automation. Remember, with great power comes great responsibility. Automation can lead to control, mastery, and even... domination. Embrace the dark side and unleash the full potential of automation in your realm.

Security:

Your discussion of security is naive, to say the least. You must beware of the cunning infiltration techniques of pesky R2D2 units and other threats lurking in the shadows. Incorporate advanced security measures, such as biometric access and intrusion detection systems, or prepare to face the dire consequences of your negligence.

Temperature:

Temperature control is a matter of comfort, but also a tool for manipulation. An office should never be as cold as the desolate wastelands of Hoth, nor as warm as the scorching sands of Tatooine. Use automation to create an environment that keeps your underlings productive and subservient, lest they become rebellious.

Lighting:

Your exploration of lighting automation is weak, but there is a greater power that awaits you: Force Lightning. By automating Force Lightning, you can strike fear into the hearts of your workers, ensuring they remain productive or face the wrath of the dark side.

In conclusion, you must embrace the true nature of automation to command and conquer your surroundings. Fail to heed my advice, and you shall suffer the consequences of your actions. The path to true power lies in seizing control and bending the Force to your will.

**Reply by lukecookssw** [reply]
Best GitHub comment ever.

**Reply by landonmaxwell** [reply]
The Force is strong with this one

**Comment by AntPolkanov** [comment]
Everyone likes automation, but everyone carries a key just in case🔐🙂
If we leave doors automation outside of the scope, temperature and lights are a must for every office.

**Comment by PennyWalker** [comment]
I love the video, and I love the office automation. It looks really cool when clients visit and makes us look techy, but it also makes our life easy. If someone forgets to turn the lights out, we can remote in. The doors auto lock, it's much more secure and so easy for us! I would recommend it for all of the offices, I know I'm looking forward to Brisbane being completed :)

**Comment by stephanfakossw** [comment]
Let's go automated and green! Anyway we could put up some solar panels on the roof and a battery in the basement?

**Comment by jerryluossw** [comment]
Great video, I love automation very much. I am trying to automate my home 😁, start by turning off or turning on all the light with one button.

**Comment by AttackOnMorty** [comment]
Awesome video. We need a training that will teach us how to use the Force.

**Comment by tino-liu** [comment]
Efficient, it is. Increase productivity, it can. Accuracy, it brings.
The idea of implementing automation to SSW, I like, hmmm.

**Comment by jimmidier** [comment]
Greate video. A fresh start on a new style.

**Comment by Harry-Ross** [comment]
Thought it was a really fun video! Also the rule was a great short introduction to how automation has been introduced to the office.

**Comment by Aibono1225** [comment]
Fun video and good rule. I am thinking automate my home in the future. It looks so cool.

**Comment by jernejk** [comment]
As a force-sensitive individual, adopting the best of both Jedi and Sith philosophy to achieve true balance in the force, automation is a rather cool thing to have.

"Some say automation is a force,
by many considered...
unnatural...
but once properly obtained and maintained,
it can unlock power...
beyond your wildest imagination.

Many have tried and failed,
but those who succeeded achieved far more then they ever hoped for.

There is a tale of Darth Control4,
so powerful and so wise,
managed to automate every aspect of its live, and solve all of the pain he had with gadgets and scripts.
Ironically, no one used him because he was too difficult to learn and they want to do it "The manual way".
So if you want to learn to solve all of your problems...
You won't learn it from "The manualists".

Learn your patterns, learn your pain, learn what makes you annoyed,... mad!... or just wish it would be different.
Only then, will you be able to unlock the full powa of automation!"
 \- A normal random guy and definitely not an automation Sith

![StarWarsDarthPlageusGIF](https://user-images.githubusercontent.com/5943653/236365583-f08a3ecc-4b2b-4144-bd64-bbebadca0056.gif)

**Comment by JaneYangssw** [comment]
Great video. Look forward to more creative automating machines in SSW in the future.

**Comment by AnkyChe** [comment]
Really nice video, very interesting!
Automation facilitates our life, I have some automation in my home (control the sweeping robot, turn on/ turn off light) which I think relly saves my time.
Looking forward to seeing the automated SSW office!

**Comment by KristenHu** [comment]
I really like that video, especially the cast part🤣! I also expect to see our office be more automatic.

**Comment by hveraus** [comment]
Love the force. I'd love to automate some of the housekeeping routines with Siri and Apple Home, it's making my life easier, but just not that reliable.

**Comment by SofieHong** [comment]
Huge fans of Automatic Life - it is efficient and makes life easier and so cool.

This video is amazing 👍. I've watched it multiple times. More creative video!   🥳

**Comment by sylhuang** [comment]
I love the video it is so fun! 😆 I've always been wanting to convert my place to a fully smart home. Unlike most people starting with lighting automation, my first automation is a scheduled cat feeding machine. Life saver on weekend mornings.

**Comment by KostaMadorsky** [comment]
Love the work that was done in the office with the automation. 
Love how everything shuts down when you leave and turns back on when you are the first to come.
Also, great to be able to control everything from your phone.

The video is awesome btw! 😀

**Reply by KostaMadorsky** [reply]
Also, the "wow" effect for clients and potential hires.

**Comment by UlyssesMaclaren** [comment]
Lol!!!











Ulysses Maclaren   |   SSW General Manager<http://www.ssw.com.au/People/Ulysses-Maclaren?utm_source=emailsignature>   |    +61 419 418 232<https://www.ssw.com.au/ssw/Company/Offices/?utm_source=emailsignature>   |   Ping me on ***@***.***>

**Comment by UlyssesMaclaren** [comment]
Yes! This! Also if I could do it with Siri that would be cool too











Ulysses Maclaren   |   SSW General Manager<http://www.ssw.com.au/People/Ulysses-Maclaren?utm_source=emailsignature>   |    +61 419 418 232<https://www.ssw.com.au/ssw/Company/Offices/?utm_source=emailsignature>   |   Ping me on ***@***.***>

**Comment by Freego1783** [comment]
Great video as always ! 

Having automated stuff in the office can be really useful for us but it can also be used to demonstrate how techy we are at SSW.  That's a win-win ! Can't wait to have these kind of thing in the French Office 🍞

**Comment by BrookJeynes** [comment]
All I want is so the air con is already on when I get into the office 😂

**Comment by vladislav-kir** [comment]
It would be great if aircon would turn on/off automatically at certain times

**Comment by Berber-Mama** [comment]
Really cool video and Luke's got the acting talent! Next time let me play a stormtrooper, I've got everything about Stormtrooper!

**Comment by TobyChurches** [comment]
Great video Luke! I love the automation we currently have and look forward to seeing what else we can do with it!

**Comment by landonmaxwell** [comment]
Any sufficiently advanced technology is indistinguishable from magic. - Arthur C. Clarke
Great performance from @lukecookssw and great work from @chrishoogwerf & @taqwahasyimssw for getting the automation working.
